system-tests/lib/cre/environment/fund_cl_nodes_op.go

```diff
@@ -263,3 +280,34 @@ func fundSolanaAddress(ctx context.Context, testLogger zerolog.Logger, node deve
 
 	return nil
 }
+
+func fundTronAddress(ctx context.Context, testLogger zerolog.Logger, node devenv.Node, fundingAmount uint64, bcOut *cre.WrappedBlockchainOutput, _ map[string]map[uint64][]byte, env *cldf.Environment) error {
+	nodeAddress := node.AccountAddr[strconv.FormatUint(bcOut.ChainID, 10)]
+	if nodeAddress == "" {
+		return nil // Skip nodes without addresses for this chain
+	}
+
+	testLogger.Info().Msgf("Attempting to fund TRON account %s", nodeAddress)
+
+	receiverAddress := address.EVMAddressToAddress(common.HexToAddress(nodeAddress))
+
+	tronChains := env.BlockChains.TronChains()
+	tronChain, exists := tronChains[bcOut.ChainSelector]
+	if !exists {
+		return fmt.Errorf("TRON chain not found for selector %d", bcOut.ChainSelector)
+	}
+
+	tx, err := tronChain.Client.Transfer(tronChain.Address, receiverAddress, int64(fundingAmount)) //nolint:gosec // G115: potential integer overflow
```

Comment:
instead check if `libc "github.com/smartcontractkit/chainlink/system-tests/lib/conversions"` (imported in this file already) has a function to handle that conversion